File: Android/Vine-5.50.1/Decompilation/app/src/main/java/co/vine/android/scribe/model/ShareDetails.java
package co.vine.android.scribe.model;

import com.bluelinelabs.logansquare.annotation.JsonField;
import com.bluelinelabs.logansquare.annotation.JsonObject;
import java.util.List;

@JsonObject
/* loaded from: classes.dex */
public class ShareDetails {

    @JsonField(name = {"has_comment"})
    public Boolean hasComment;

    @JsonField(name = {"message_recipients"})
    public List<VMRecipient> messageRecipients;

    @JsonField(name = {"post_id"})
    public String postId;

    @JsonField(name = {"share_targets"})
    public List<String> shareTargets;
}

File: Windows Phone 8.1/2.5.7.0/Decompilation/Vine.WindowsPhone/Vine.Models.Analytics/ShareDetails.cs
using System.Collections.Generic;
using Newtonsoft.Json;

namespace Vine.Models.Analytics;

public class ShareDetails
{
	[JsonProperty("share_targets")]
	public List<ShareTarget> ShareTargets { get; set; }

	[JsonProperty("has_comment")]
	public bool HasComment { get; set; }

	[JsonProperty("post_id")]
	public string PostId { get; set; }

	[JsonProperty("message_recipients")]
	public List<VMRecipient> MessageRecipients { get; set; }
}
